continuous_integration/travis/run_tests.sh

```diff
@@ -13,7 +13,3 @@ else
     echo "py.test dask --runslow $XTRATESTARGS"
     py.test dask --runslow $XTRATESTARGS
 fi
-
-# This needs to be enabled to test __array_function__ protocol with
-# NumPy v1.16.x, enabled by default starting in v1.17
-export NUMPY_EXPERIMENTAL_ARRAY_FUNCTION=1
```

Comment:
Why are we removing this?  Is the entry in `.travis.yml` sufficient?  cc @pentschev

Reply:
This `export` is causing the Travis builds to pass regardless if there are failed tests (see https://github.com/dask/dask/pull/4525#issuecomment-473414348 and the few comments that follows). The entry in `.travis.yml` should be sufficient. This export was, I think, accidentally added in #4567 and is currently being removed in #4525.

Reply:
Now that I've typed that out, it makes sense to get this `export` removed in `master` ASAP so we don't unintentionally merge in failing tests. I can open up a separate PR that removes `export NUMPY_EXPERIMENTAL_ARRAY_FUNCTION=1` in case this PR or #4525 take some time to get merged.

Reply:
Opened up #4605

Reply:
That's correct. The entry in `.travis.yml` should be sufficient. The `export` should have been removed and was indeed the cause for CI not to fail.

Reply:
Sorry for that, it was my fault that this `export` ended up there and caused builds to fail.

Reply:
Don't sweat it, these things can happen. I'm glad you were able to identified the issue so quickly :)